app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -147,6 +161,18 @@ async def raster_tile_cache_asset(
         )
         job_list += symbology_jobs
 
+        # FIXME: Surely this could be done better.
+        # Maybe make a lookup table of bit depths for symbology types?
+        # The larger question is should we infer the bit depth from the
+        # symbology (or even from the data in the source) or give people
+        # the flexibility/responsibility of specifying it?
+        # Doing this for the moment as it's the only case in which we
+        # want 16-bit PNGs
+        if symbology_function == date_conf_intensity_multi_16_symbology:
```

Comment:
Agreed, why don't you just create a lookup around where you define the symbology constructor lookup?

Reply:
Done.

Reply:
Done